### PR TITLE
Extended CKBuilder to support ESM license header

### PR DESCRIPTION
**Pull request by jacekbogdanski**
I've added an additional option to support LTS license headers.

### CODE REVIEW

File: src/lib/tools.js [hunk 1]
```diff
@@ -107,8 +107,19 @@
...
+			if ( CKBuilder.options.lts ) {
+				copyright = 'CKEditor 4 LTS ("Long Term Support") is available under the terms of the Extended Support Model.';
+			} else if ( CKBuilder.options.commercial ) {
+				copyright = 'This software is covered by CKEditor Commercial License. Usage without proper license is prohibited.';
+			} else {
+				// Copyright only needs to be updated for LTS or commercial license.
+				return;
+			}
 
+			// `lineEndings` array includes all filetypes supported by CKEditor 4.
```
Comment (jacekbogdanski) [inline src/lib/tools.js:122]: It seems like this method is only called for files that had updated line endings, for example language files. I'm not sure if that's the design decision or some oversight - but the binaries of CKEditor 4 should have properly updated license headers for LTS, so I didn't change the original behavior of this script (I've only added support for LTS).

**Review by vokiel — CHANGES_REQUESTED** [review]
It would be great to have this option also descirbed in the help, e.g. https://github.com/ckeditor/ckbuilder/blob/4ff7ff6f2cac8908d7b1c16f84ec04ada7661553/src/assets/help.txt#L27

**Comment by jacekbogdanski** [comment]
@vokiel thanks, added 👍🏻

**Comment by vokiel** [comment]
Tested, it works. Adds proper license header to bilt `ckeditor.js` and to translation files.

But only to them. Other files with copyright header still contains the default:

```javascript
/**
 * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
 * For licensing, see https://ckeditor.com/legal/ckeditor-oss-license
 */
```

**Pull request merged**